test/supervisor.js

```diff
@@ -145,6 +145,21 @@ describe('supervisor', function(done) {
     });
   });
 
+  describe('log decoration', function() {
+    var TAGS_WORKER = /^pid:\d+ worker:\d+ .+/;
+    var TAGS_SUPER = /^pid:\d+ worker:0 .+/;
+    var NO_TAGS = /^.+/;
```

Comment:
can you look for some message from the test app, and verify its at start-of-line? `/^yes/` looks like it might work.